docs/ConfigureHPCC/CMakeLists.txt

```diff
@@ -25,7 +25,7 @@ ENDFOREACH()
 # commented out per issue # HPCC-10267
 #
 #
-DOCBOOK_TO_PDF( ${FO_XSL} UsingConfigManager.xml "UsingConfigManager")
+DOCBOOK_TO_PDF( ${FO_XSL} ConfigureHPCC.xml "ConfigureHPCC")
```

Comment:
If the title is "Configuring the HPCC System", then the filenames should be ConfiguringHPCC (both the XML file and the output file). the folder should also be ConfiguringHPCC